unlock-app/src/__tests__/services/walletService.test.js

```diff
@@ -55,15 +57,75 @@ nock.emitter.on('no match', function(clientRequestObject, options, body) {
 })
 
 let walletService
+let config
 let providers
 
 describe('WalletService', () => {
+  const WalletService = require('../../services/walletService').default
```

Comment:
I think this really belongs in the top of the file?

Reply:
The loading of promises.js has to happen after the `jest.mock('../../utils/promises')` so we can't use import, but have to use require. It's awkward but that's the price of module mocks :)

Reply:
What? let's debug this tomorrow because I have been using mocks all over the place and never had to do that ;)
jest will actually hoist the mock for modules at the beginning of files no matter what.